corehq/apps/users/tests/test_role_utils.py

```diff
@@ -118,3 +125,75 @@ def _delete_presets(self):
         for role in UserRole.objects.get_by_domain(self.domain):
             if role.id != self.role1.id:
                 role.delete()
+
+
+class TestCommcareAnalyticsRolesByUser(TestCase):
+
+    DOMAIN = "domain1"
+    USERNAME = "username"
+    PASSWORD = "***"
+
+    @classmethod
+    def setUpClass(cls):
```

Comment:
You need to call the superclass's setUpClass method, otherwise database setup won't get called.

```suggestion
    def setUpClass(cls):
        super().setUpClass()
```